text/0000-advance-security-disclosure.md

```diff
@@ -0,0 +1,305 @@
+  not originating from a fork of the toolchain provided by the Rust project.
+  The implementation has to be mature, widely used and must follow all policies
+  on alternate implementation set by the Rust project (if any are set).
+
+Note that when evaluating the eligibility criteria, the spirit of the rules
+will be considered: if the WG thinks an organization that would not be eligible
+otherwise is trying to find a loophole to be able to apply, the WG will have
+the authority to reject the application.
+
+Organizations will be able to apply by opening an issue in the
+[rust-lang/wg-security-response] repository, detailing:
+
+* Which of their products or projects would make them eligible to be notified.
+
+* One or more email addresses that should receive the advance disclosure.
+
+* Two or more people that will serve as the organizational contact: those
+  people will be contacted by the WG whenever the WG needs to discuss something
+  with the organization.
+
+* A pledge to maintain confidentiality, as detailed in the "Confidentiality"
+  section of this RFC. The person submitting the application must be authorized
+  to enter that pledge on behalf of their organization.
+
+The application process will be public, and the decision will be made by the
+WG: a majority of the WG members needs to be in favor of the organization
+joining, and none of the members needs to be against it. The resulting list
+of organizations receiving advance disclosure will also be public.
+
+In addition to individual organizations, industry-standard mailing lists and
+disclosure venues that are frequented mostly by organizations that would
+otherwise be eligible to be notified can be added to the list, if the WG
+determines the venue is appropriate, and has sufficient security and
+confidentiality practices.
+
+Organizations won't need to pay any fee to receive early notifications, nor
+will they have to be a sponsor to the Rust Foundation. There will be no formal
+NDA in place, only the informal confidentiality pledge.
+
+Once the application is accepted, the organizational contacts will be able to
+request changes to the address receiving the notifications or the list of
+contacts by opening another issue.
+
+### Removal from the list
+
+Organizations can be removed from the list in any of these cases:
+
+* The reason why the organization joined the list is not applicable anymore,
+  for example if they discontinued their distribution. Organizations can
+  re-apply later if they become eligible again.
+
+* The organization did not act on the early notifications it received for an
+  extended period of time. The WG will work with the organization to see if
+  there are valid reasons for not acting on the notifications before removing
+  the organization. Organizations can re-apply later.
+
+* The organization either accidentally violated confidentiality multiple times
+  without taking appropriate steps to prevent it from happening again, or
```

Comment:
```suggestion
* The organization either accidentally violated confidentiality more than once,
  without having taken appropriate steps to prevent it from happening again after the first time, or
```

```suggestion
```

Trying to be more explicit here about the point at which we expect people to take appropriate steps to prevent mistakes from happening again.